fe/fe-core/src/main/java/org/apache/doris/analysis/PrepareStmt.java

```diff
@@ -133,22 +144,32 @@ public ByteString getSerializedOutputExprs() {
 
     @Override
     public void analyze(Analyzer analyzer) throws UserException {
```

Comment:
should support nerids planner

Reply:
https://github.com/apache/doris/pull/22167 skip process prepare command by Nereids